plugins/ezid/management/commands/register_ezid_doi.py

```diff
@@ -77,11 +78,13 @@ def handle(self, *args, **options):
         #debug breakpoint, use to confirm the metadata gathered above
         # pdb.set_trace()
 
-        ezid_config = {'shoulder': SHOULDER,
+        ezid_settings = RepoEZIDSettings.objects.get(repo=repo)
+
+        ezid_config = {'shoulder': ezid_settings.ezid_shoulder,
                        'username': USERNAME,
```

Comment:
It would be a nice refactor to add these current ENV configs to the ezid_settings config as well, thoughts?